scripts/lld.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env sh
+
+TEXT_BOLD=$(tput bold)
+TEXT_NORMAL=$(tput sgr0)
+
+LD_LLD_WITH_VERSION=$(command -v ld.lld-$1)
+LD_LLD_WITHOUT_VERSION=$(command -v ld.lld)
+
+if [ -z $1 ]; then
+    echo "No version specified!"
+    echo "Usage:   $0 <LLD version>"
+    echo "Example: $0 13"
+    exit 1
+fi
+
+# Check if the specified `lld` version is present on the system
+if [ -z $LD_LLD_WITH_VERSION ]; then
+    echo "Could not find ld.lld-$1, make sure lld is installed"
+    exit 1
+fi
+
+# Check if `ld.lld` is present on the system
+if [ -z $LD_LLD_WITHOUT_VERSION ]; then 
+    # Not present, create a symlink into ~/.local/bin
+    mkdir -p $HOME/.local/bin
+    ln -svf $LD_LLD_WITH_VERSION $HOME/.local/bin/ld.lld
+
+    echo "\nYou might need to update your \$PATH environment variable for cargo to recognize lld, like so:"
+    echo "${TEXT_BOLD}echo 'export PATH=\$HOME/.local/bin:\$PATH' >> $HOME/.bashrc${TEXT_NORMAL}"
```

Comment:
I know this is not popular under Linux users but would it be also OK to append the newly created `PATH` variable without the users consent? When installing Rust for example the following line `. "$HOME/.cargo/env"` is appended without consent.

Reply:
Yes, updating the PATH in .bachrc the way cargo does is fine i think. It will only work if you have bash but that's majority of people

Reply:
Cool, I went ahead and implemented it that way; it's more convenient this way because the user doesn't have to do anything other than executing the script :)